src/meta/app/src/schema/table.rs

```diff
@@ -281,6 +281,14 @@ pub struct TableIndex {
     // if true, index will create after data written to databend,
     // no need execute refresh index manually.
     pub sync_creation: bool,
+    // if the index columns or options change,
+    // the index data needs to be regenerated, 
+    // version is used to identify each change.
+    pub version: String,
+    // index options specify the index configs, like tokenizer.
+    pub options: BTreeMap<String, String>,
+    // refresh time, the most recently refreshed index will be used first.
+    pub refreshed_on: Option<DateTime<Utc>>,
```

Comment:
What's the purpose of the `refreshed_on`? It appears to be unused in the code.

Reply:
In the binder stage, if there is more than one index available for this table, we will prioritize the most recently refreshed index to ensure that index data is available. This logic will be added later.